### PR TITLE
PRIME-2254 (PRIME-2238) HA Static Text Changes

### PR DESCRIPTION
**Pull request by bergomi02**
HA Static Text Changes

https://pr-2179.pharmanetenrolment.gov.bc.ca/health-authority

### CODE REVIEW

**Review by Paahn — CHANGES_REQUESTED** [review]

File: prime-angular-frontend/src/app/shared/components/auth/site-reg-access/site-reg-access.component.html [hunk 1]
```diff
@@ -68,7 +68,7 @@ <h2>BC Services Card</h2>
           <h2>Who can register a site?</h2>
 
           <p>
-            The person legally responsible for an organization’s PharmaNet use registers the site.
+            A person authorized by the health authority can registers the sites.
```
[the remark this answers is not in the document]
Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/components/auth/site-reg-access/site-reg-access.component.html:71]: good catch.

Comment (Paahn) [inline prime-angular-frontend/src/app/shared/components/auth/site-reg-access/site-reg-access.component.html:71]: text change?

**Review by Paahn — APPROVED** [review]
LGTM

File: prime-angular-frontend/src/app/shared/components/auth/site-reg-access/site-reg-access.component.html (hunk 1 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/components/auth/site-reg-access/site-reg-access.component.html]: my bad, the change was not pushed.

**Pull request merged**